src/tools/x/src/main.rs

```diff
@@ -72,15 +69,15 @@ fn main() {
         let candidate = dir.join("x.py");
 
         if candidate.exists() {
-            let mut python = Command::new(python());
+            let mut cmd = x_command();
 
-            python.arg(&candidate).args(env::args().skip(1)).current_dir(dir);
+            cmd.arg(dir.join(X_SCRIPT)).args(env::args().skip(1)).current_dir(dir);
```

Comment:
If you pass in `dir` to `x_command`, you can get rid of the cfgs for `X_SCRIPT` too :)